src/components/Chat/BottomSheetAttachment.js

```diff
@@ -23,7 +23,7 @@ const BottomSheetAttachment = React.forwardRef((props, ref) => (
             <Text style={styles.textCard}>Send Photos & Videos</Text>
           </View>
           {props.isLoadingUploadMedia ? (
-            <ActivityIndicator size="small" color={COLORS.white2} />
+            <ActivityIndicator size="small" color={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `ActivityIndicator` has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change is intentional and that the new color provides sufficient contrast against the background for accessibility purposes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->